packages/manager/modules/pci/src/projects/project/storages/databases/databases.html

```diff
@@ -55,6 +55,11 @@
             data-sortable
             data-filterable
         >
+            <img
+                width="50px"
```

Comment:
Hard coding the width of an image might not go well with the mobile view. Have it in terms of rem

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1